io/src/main/scala/aqua/files/Imports.scala

```diff
@@ -0,0 +1,56 @@
+package aqua.files
+
+import fs2.io.file.Path
+import scala.util.Try
+
+/**
+ * Imports resolution configuration.
+ */
+final case class Imports(
+  settings: Map[Path, Imports.PathSettings]
+) {
+
+  /**
+   * Get all possible resolutions for a given import.
+   *
+   * @param from path of the file that imports
+   * @param imported import string
+   * @return list of possible resolutions
+   */
+  def resolutions(from: Path, imported: String): List[Path] =
+    relative(from, imported).toList ::: gather(from, imported)
+
+  // Return relative resolution if possible
+  private def relative(from: Path, imported: String): Option[Path] =
+    for {
+      fromParent <- from.parent
+      importedPath <- Try(Path(imported)).toOption
+    } yield fromParent.resolve(importedPath)
+
+  // Gather all possible resolutions from imports config
+  private def gather(from: Path, imported: String): List[Path] =
+    // First - find the longest matching prefix for path
+    settings.filter { case (prefix, _) =>
+      from.startsWith(prefix)
+    }.maxByOption { case (prefix, _) =>
+      prefix.toString.length
```

Comment:
Shouldn't the longest prefix be compared by the number of folders, not characters?

Reply:
If two paths are prefixes of `from`, the longer one have at least the same amount of folder as the shorter one. 
Also, they can have the same amount of folder, example:
`from = /a/b/ccc/d.aqua, p1 = /a/b/c, p2 = /a/b/cc`, `p1` and `p2` have the same amount of folders. So we should pick `p2` because it is a longer prefix.